oak-lucene/src/main/java/org/apache/jackrabbit/oak/plugins/index/lucene/writer/IndexWriterUtils.java

```diff
@@ -70,6 +73,10 @@ public static IndexWriterConfig getIndexWriterConfig(LuceneIndexDefinition defin
             IndexWriterConfig config = new IndexWriterConfig(VERSION, analyzer);
             if (remoteDir) {
```

Comment:
one reason why remoteDir could be true is because copyOnWrite is disabled (which usually isn't the case).

So effectively the new code change here (the else block) will impact normal indexing as well (not just the out of the band indexing from oak-run), from what I can tell - was that taken into account ?